extensions/amp-geo/0.1/amp-geo.js

```diff
@@ -219,25 +219,26 @@ export class AmpGeo extends AMP.BaseElement {
   }
 
   /**
-   * checkGroup_() does this.country_  match the group
+   * checkGroup_() does this.country_ match the group
+   * after expanding any presets and forceing to lower case.
    * @param {!Array<string>} countryGroup The group to match against
    * @return {boolean}
    */
   checkGroup_(countryGroup) {
-    countryGroup = countryGroup.reduce((countries, country) => {
+    /** @type {!Array<string>} */
```

Comment:
Nit: Local variable typecasts are unnecessary in 99% of cases.

Reply:
It complained when I didn't typecast. :-(

Reply:
Hm, guess it's the 1% of times it's needed.